SweetTweetApp/src/main/webapp/resourses/js/View/View.js

```diff
@@ -0,0 +1,54 @@
+class View {
+    addPostsToNewsLine(posts, currentAuthor){
+        let postsWrapper = document.getElementById("news-line");
+        postsWrapper.innerHTML = "";
+        posts.forEach((post) => {
+            const f = this.renderPost(post, currentAuthor);
+            postsWrapper.appendChild(f);
+        });
+    }
+
+    renderPost(post,currentAuthor) {
```

Comment:
In general, I liked your idea of using templates. In the future, it would be great to generalize it a bit to render not just post, but pass something like template name and all the required props to be rendered 👍 . I understand, it not so easy because of some conditions inside, but still